### PR TITLE
Add remodel of the Bactrian & Mule

### PR DESCRIPTION
**Pull request by Saugia**
**Content (Artwork)**
## Summary

This PR replaces the current Bactrian model and the current Mule model with newer up-to-date designs. The newer models are more fleshed out as a whole ship (more base to the shape, rather than _only_ blocking it together), add a bit more bulkiness, and add more details to reflect ships such as the newer Mod Argosy, Dreadnought, and their details.

Comparison
thumbv3
topv3
combatv3

The models add window structures to more attribute to the City-Ship look (which, if you really want to get deep, represent the two humps of a camel as the Bactrian is named after.) The fins are decreased in size to visually take less space of the design itself, and that space is used for more bulkiness, and more fins are added for a better visual representation of scale. 

## Save File
This save file can be used to play through the new mission content:
Should be playable if you have the ships unlocked or captured.

## PR Checklist
 - [x] I updated the copyright attributions, or decline to claim copyright of any assets produced or modified
 - [x] I uploaded the necessary image, blend, and texture assets here: https://drive.google.com/drive/folders/1E_p8CRpBGOEcjL9jQsD-efrMkjEeWyAT?usp=sharing
 - [x] I created a PR to the [endless-sky-high-dpi repo](https://github.com/endless-sky/endless-sky-high-dpi) with the `@2x` versions of these art assets: (https://github.com/endless-sky/endless-sky-high-dpi/pull/204)

### CODE REVIEW

**Comment by JanErikAhrens** [comment]
@Amazinite may I ask why you downvoted? If you could at least give your reasons that would be appreciated, because I really don't think it's fair.

**Comment by Terin** [comment]
Yep, a thumbs up requires no action on the PR's author, while a thumbs down does. Leaving it with no feedback is actively unhelpful.

**Review by Zitchas — APPROVED** [review]
I must admit that this is probably one of the redesigns that I think has the most value. This doesn't just give it a bit more detail, it actually gives it *weight* and substance. This new sprite makes the city ship really feel like a true city, like the hefty ship that it is.

Likewise, moving the gun ports to the two angled batteries is probably one of the best uses of the angled guns that we could hope for: It makes those gunports both less useful for sticking guns in, and it makes them *more* useful for spreading out missile fire to take out fighters & light craft that might be harassing it. This is a clear win-win as far as I'm concerned. 

I know I've argued that the Bactrian should have *fewer* guns in the past. Having the gun ports moved to these angled positions, I think, actually resolves my issues with the Bactrian, as this makes it clear that these are missile ports, which are highly suitable to the role, lore, and description of the ship; and not gun ports, which are ridiculous and unsuitable on a city. 

Also, from an aesthetic view, having guided munitions launch out at an angle and arc around to hit things looks really cool.

**Comment by Zitchas** [comment]
My only iffy point would be whether or not it should be two torpedoes together on one side, with two missiles on the other; or one of each on each side. The former makes more sense to me from a logistics perspective (keep the stores of each projectile together), but the second makes more sense from a "spread the fire" perspective to ensure that there's a missile being launched from each side for maximum anti-light-craft potential. But I'm 50/50 on which set up I prefer, so this is just a comment.

**Comment by vitalchip** [comment]
Definitely putting this in at 2x size.

**Comment by McloughlinGuy** [comment]
It looks so much better than the current bactrian, and it fits the art style of endless sky. I like it a lot.

**Comment by WarlordMike** [comment]
The Baccy always struck me as one of those ships that seemed like any attempt to revise the design would be a fruitless endeavor.

I've never been so happy to be so _thoroughly_ proven wrong. She deserves this makeover, Saugia.

**Comment by LepRyot** [comment]
phattest ship ever seen !!
Looks like you accomplished a long standing goal for a lot of people

File: data/human/ships.txt [hunk 1]
```diff
@@ -332,22 +332,32 @@ ship "Bactrian"
...
+		angle 0
+		under	
+	gun -67.5 65 "Sidewinder Missile Launcher"
+		angle 305
+	gun -62.5 56.5 "Sidewinder Missile Launcher"
+		angle 305
+	gun 67 -126 "Torpedo Launcher"
+		angle 40
+	gun 74 -120 "Torpedo Launcher"
+		angle 40
```
Comment (Amazinite) [inline data/human/ships.txt:351]: I'm not a fan of using angled gun ports here. It's an idea that I've discussed in the past for alien ships, but I'd rather not have angled gun ports on any human ships, reason being to keep the more interesting mechanics for later in the game instead of showing off everything in human space.

**Review by Amazinite — CHANGES_REQUESTED** [review]
I'm not too sure on the bigger, bulkier look for the Bactrian. A ship's appearance should fit its stats, and the Bactrian is already the biggest ship in human space, and (ignoring the Kestrel, which is a bit of a special case) the second toughest ship in terms of HP, behind only the Carrier (26,100 vs 29,700). The Carrier being smaller but tougher could perhaps be chalked up to the Carrier using tougher materials, but this remodel makes the Bactrian *even bigger*, to the point where I'd argue that it doesn't really fit the stats anymore, at least in terms of HP. 
Put another way, something that's been mentioned in the past is the "rule of cool," i.e. the cooler something looks the more powerful it should be. So in a way, this bigger, bulkier Bactrian, at least at its current size, is almost too cool for its own good. Zitchas says this image would make him stop wanting the Bactrian to be nerfed, but for me it's to the point where you'd actually make me want to buff the Bactrian, which I never thought would be the case.
Untitled
I imagine if you downscale the ship a bit so that it at the very least isn't bigger length-wise than the existing Bactrian that that would help.

Aside from that, the post processing on the ship image leaves certain areas of the ship looking rather dark, but that same post processing isn't done on the thumbnail, making the thumbnail much brighter than the ship image. This same thing can actually be seen in the old images too to some extent, but it's not nearly as drastic. Not a massive issue and it might be fine to let it slide, but it is something that I noticed.
Untitled

You could also trim the height on the thumbnail and width on the ship image.

File: data/human/ships.txt (hunk 1 above)
Comment (Zitchas) [reply data/human/ships.txt:345]: This would make considerable sense if we were discussing, say, putting slowing weapons on a Bactrian. However, this is not what we are discussing. We are discussing gun-angles, which should not be considered a "special feature", but rather a standard element of ship design to make our current often-overly-similar ship designs more varied.

In the past I have highlighted two facets of the Bactrian that have generally been contradictory:
1. As a *city*-ship, as well as being a ship of its size, guns are inappropriate armament. It should *never* be spinning around fast enough for guns to be a meaningful weapon; even if our current outfitting system means that it frequently happens.
2. The Bactrian's lore emphasizes its role as being a ship used to venture into unsettled territory to establish and supply colonies. Its defensive capabilities are justified as being there to protect it from ambushes while enroute.
3. The Bactrian's stock loadout for the guns revolves around homing projectiles. As a general principle, homing projectiles are ideal ordinance for a heavy ship looking to take down agile attackers. 

Now, ambushes, by general definition, happen from inopportune (to the defender) angles: Coming in from hyperspace behind them, around planets or large asteroids to the sides, etc. Virtually by definition ambushes and raids are things that come from the sides, while *blockades* are opposition from the front. Nowhere is the Bactrian described as being a blockade runner. As such, virtually all the combat it is expected to be engaged in involves combatants coming at it from the sides or rear. Further more, given the size of the ship, it is virtually guaranteed that it is facing ships that are considerably more agile than it.

There is also the problem that, in its present vanilla configuration, Bactrians are overly good in my opinion at being warships. This is, in large part, due to the number of guns that can be fitted with heavy lasers or other high powered guns. I have in the past PR'd changes to reduce the number of guns, which have been denied. I'm sure there were a variety of reasons, but one of the ones I recall was the fact that having a large number of missiles weapons was, in fact, both perfectly reasonable and in keeping with the lore for this ship.

In this PR we have something that works to strengthen and satisfy both these elements:
3. Having angled gun placements starts off projectiles at a 45 from the Bactrian's forward axis, which gives projectiles a good head start in their turn to strike at ships that are anywhere but in front of it.
2. Given the original design expecting to counter ambushes, having the ports setup this way maximizes the offensive potential of the ship against ambushers and pursuers.
1. This means that the Bactrian doesn't need to spin nearly as much to bring its weapons to bear. As such, it can focus on getting to its destination or using that outfitting space for other things.
0. It also reduces the suitability of using these gunports for guns. I'm sure some people will still find creative flight profiles that make good use of them for enabling a strafing style attack pattern;  but for the most part people will simply skip putting guns in them and utilize homing projectiles as intended. This reinforces the previous three points.

In other words, having these guns angled as they are makes them more in line with their lore, more effective in their intended usage as homing projectile launchers, and less effective in their non-intended use for guns., which in turn reduces the need for Bactrians to be able to turn quickly since none of their weapons depend on it.

In my view, this makes this change a win for each of our perspectives, a win for the Bactrian's lore, and a win for demonstrating to our community what the game engine can actually handle. Given that even most of the people on Discord haven't yet even realized that angled gun are at thing. What better place to showcase it than on an eminently sought after ship like this?

Comment (Zitchas) [reply data/human/ships.txt:345]: The discord poll is less than 24h old, so I'm not sure everyone's gotten a chance at it. That being said, the vote tally is currently 339 to 31 in favor of the angled gun ports. While obviously that doesn't change the fact that the Devs get veto, I'd say this is a prime situation where we can follow the Kestrel (guns) experiment: Keep the angled guns in for the Bactrian for now, and see how things go. In a few months after everyone's had a chance to actually *experience* how they play in the course of regular gameplay, we re-evaluate. 

In any case, my interpretation of the discussion is that the community is firmly on the side of "gun angle is a basic feature of ship design, not something fancy for future factions" line of thought. Not unanimous, but pretty strong.

Comment (Arachi-Lover) [reply data/human/ships.txt:345]: The poll has 4 points supporting the angled gun ports, written by the one who made the poll and is in favor of them, and a very short summary of points against it that were not even written out or developed in this PR yet, instead of letting the person against the angle ports write their own points and have those listed out just as the arguments for having these angled ports are.

Whether that'd change the results of the poll or not, it's still a very bad practice to create a poll that'll of course be biased when one presents several arguments for one side and a diminutive summary of the other.

Comment (MCOfficer) [reply data/human/ships.txt:345]: Valid point, but at 10:1 in favour, it's probably negligible.

Comment (Amazinite) [reply data/human/ships.txt:345]: Debatable.

Comment (Zitchas) [reply data/human/ships.txt:345]: There was one point given against it. Still is only one point against it. I would assume that if someone is going to list only a single reason to oppose something, I would expect them too pick their strongest point. In any case, the community seems pretty unimpressed with the idea of saving gun angles as an "advanced" thing for higher tech races/factions.

Comment (MCOfficer) [reply data/human/ships.txt:345]: So, how do we resolve this? Ask again with a neutral text? Disregard the poll in this discussion?

Comment (Arachi-Lover) [reply data/human/ships.txt:345]: It's not a fair poll, so I don't see why it should be taken into account. I expect a fair poll to still favor the angled gun ports, but haven't had a fair poll to know.

Comment (Zitchas) [reply data/human/ships.txt:345]: And how do you define a fair poll? Limit the pro side to a single argument? I prefer to present all the arguments, and if one side has more than the other, then that is how the cookie crumbles. The question is what do we think makes the game better, not how to ensure both sides get an even 50% of the text in the poll.

Comment (Arachi-Lover) [reply data/human/ships.txt:345]: I never said we should limit the arguments to a specific number, I never said both sides needed to have the same amount of arguments. I stated the fact that the poll is unfair because one side has its arguments there and the other doesn't, it has a tiny summary of what may be just one of that side's arguments.
You didn't give the other side time to give their arguments, you just rushed a poll out before the other side's arguments were given.
Caring for what makes the game better involves listening to what the other side has to say, and if we're relying on the community to decide that then we should wait for the other side to say what it has to say before summarizing the other side in one line while we give ourselves the luxury of presenting our arguments in full, in the poll that we created.

Comment (Zitchas) [reply data/human/ships.txt:345]: That is assuming there is anything else. Amazinite is Dev. Stuff can and does get shutdown based on his opinion alone. Since nothing more substantial was forthcoming, I made the poll based on the complete available arguments. I believe, and seems the community largely agrees with me, that his stated opinion in this case is insufficient justification.

The fact that nothing else has been said also suggests this.

Comment (Arachi-Lover) [reply data/human/ships.txt:345]: And that's assuming there is nothing else.

Comment (HTMLPhoton) [reply data/human/ships.txt:345]: Looking through the poll, I still think it's biased. One could have at the very least tried to keep the 2 arguments to equal size, by taking derpy's entire comment against a shortened version of Zitchas's.
Derpy's point also is not invalid. We know that people can get swayed a lot by first impression, and I did, on first impression, dismiss derpy's point. which I now see as perfectly valid. the 10:1 ratio *could* have been caused by the arguments presented being very lopsided.

Comment (Zitchas) [reply data/human/ships.txt:345]: This reminds me of the climate change "debate." 99.9% of climate scientists firmly endorse the science of climate change, and have for decades. But many news outlets tried to present it fairly and gave equal coverage to both "pro" and "con" information on climate change. Many people thus erroneously believed it was actually a disputed thing. 

If one side has many arguments for it, and the other side has one argument, all that means is that one side doesn't have much going for it. It doesn't make things less fair.

That being said, definitely, my framing of the poll wasn't done in the most impartial tone. But limiting it to one vs one argument is artificially inflating a position that, as of yet, still has only a single argument for it.

File: data/human/ships.txt [hunk 2]
```diff
@@ -2420,15 +2424,17 @@ ship "Mule"
...
+	engine -7.5 129
+	engine 11 128
+	turret 19.5 -40
+	turret -22.5 -54
+	turret 20 32
+	turret -20.5 103
+	gun -33.5 56.5
+		angle 305
+	gun -38.5 62
+		angle 305
```
Comment (Amazinite) [inline data/human/ships.txt:2436]: Bruh moment.

Comment (Saugia) [reply data/human/ships.txt:2436]: If it needs to be changed I don't mind. I'm just waiting out how things turn out before I commit any huge changes; that was something I already had set up to be in line with the first iteration

File: data/human/ships.txt (hunk 1 above)
Comment (Amazinite) [reply data/human/ships.txt:345]: Just because I provided a single point against using angled gun ports on the Bactrian doesn't mean that I only have a single point. You're correct in stating that given only a single point, someone is going to put their strongest point forward, but I'd say it's also important to note that how strong or persuasive a point is is going to change from person to person.

I provided the point of gating novel and unique mechanics behind later content because to me that is a very important point. It is me coming at this from a game design perspective, looking at the game as a whole and how we spread out content and ideas. I imagine though and it's perhaps evident that that is not the most persuasive point generally, at least when stacked up against a multitude of points going the other way, and I fully understand why that's the case as most people likely aren't coming at this from an overall game design perspective. I imagine for most people they're looking at it more from a "Oh that sounds cool, I like it," type of thing; very surface level. Now that isn't a jab, that's just the reality that most people aren't spending a significant amount of time formulating their thoughts on the finer aspects of game design for some random open source indie game they play from time to time.

So given that, I'm not providing my most persuasive point(s) for a general audience when reviewing a PR, I'm going to provide the point that I feel the strongest about, as reviews are intended for the creator of the PR and the point that I feel the most strongly about is what is the biggest hurdle for getting a PR merged and therefore what the creator of the PR should focus on. I could provide every point behind why I disagree with something when I review, but honestly I already spend a rather significant amount of time formulating reviews as is, particularly when there's some sort of strong disagreement involved and I want to carefully word my review (I'd estimate that the review for this PR took maybe close to an hour to write [and this post has taken even longer]; granted that was with me throwing together images in GIMP to show examples of my point and I was speaking in Discord at the same time, but it's still a large chunk of time for even what looks like a smaller review), and listing every point I have just becomes a waste of time for the PR creator to read through given that just my strongest held point against something should be enough, at least for an initial review; if the PR creator isn't convinced by my strongest held point then I can provide more after that, but from what I understand, and correct me if I'm wrong, but @Saugia doesn't have a particularly strong opinion on the state of the gun ports one way or the other, so listing every point I have would just be an exercise in convincing everyone else.

Given the storm that's been kicked up around this though I suppose I can provide more points, although I will admit that a lot of these have already been described by others on Discord, so it's not like the arguments weren't out there. In no particular order:
1) Given that what we are doing is suggesting a change for existing content, I'd say that the burden is on the people advocating for the change to convince me why the change to existing content is good. If this PR were adding a new ship to the game that used angled ports intended only for missiles then I would take less issue with it; if it were a human ship I'd still not like it, but if it were an alien ship of some sort, particularly outside of the Hai, then I might be perfectly fine with it.
2) Angled gun ports *restrict* the customizability of the Bactrian, which I do not think we should do. By angling the gun ports, the only viable guns that could be installed on the Bactrian are weapons with homing. Excluding the smaller human missile launchers since they basically just inflate the numbers (there are five of them if you want to include them), there are currently 43 weapons that a Bactrian could install in its gun ports. By angling the ports, you're restricting that to 14. (For this reason, I'd find it difficult to take any existing ship and convert any forward facing gun ports into angled gun ports, since you're removing options from the player, not adding them.)
3) I do not see the stock fitting of a ship as being justification for changing it such that only fittings like the stock fitting are viable. That is to say, just because the Bactrian comes stock with missiles doesn't mean that the Bactrian should only ever be capable of using missiles in its gun ports.
4) There have been suggestions that these angled gun ports could still be used with normal guns, which I'd contest simply isn't the case. I personally play with the auto aim setting on, and I imagine that most people do as well. What the auto aim does is if your ship is facing close enough to a target, the game automatically turns your ship to face the target, but should your gun ports be angled, this means that the game is aiming your gun ports off of the target, and given that the suggested angles aren't particularly far away from facing forward, this will occur often. So should you want to use normal guns in an angled port, either you need to have auto aim off or you need to not target the ship you are attacking, which means you're depriving yourself of information about your target ship (unless you have the overlay active, but I don't use that, and at that point you're talking about basically needing to use specific settings to make a loadout viable, which I don't think is a situation that we should be creating), and in both cases you need to manually aim your guns, which leads to human inaccuracy that's going to hurt your combat capabilities. So angled gun ports with normal guns simply can't be used normally like forward facing gun ports with normal guns.
5) Another suggestion for how angled ports could be used with normal guns is by doing "drive-bys" where you fly past a ship while firing your guns, but this simply isn't an effective tactic given that the time that your weapons will actually be on target will be so miniscule compared to if you simply had your guns facing forward and were facing your target head on or were using turrets that actually track a target as you fly past it. You'd need very particular types of weapons with high burst damage to actually be effective with this drive-by tactic, but those types of weapons simply don't exist, at least not in large numbers; you might be able to get away with it with a Nullifier or Detainer, but that's moreso because of the special effects those weapons have, and that's only two more weapons that could viably be used in angled ports. So insofar as we lack such weapons, particularly in human space, and no I wouldn't be interested in creating such weapons just to justify this one ship having angled ports, I don't find the drive-by tactic idea convincing as for how angled gun ports wouldn't hurt customization.
6) Going off of the auto aim idea that auto aim actually forces these gun ports into bad positions for using normal guns, the same is actually also true of missiles. While it is true that should a target be behind the Bactrian, angling the gun ports outward somewhat would allow the missiles to more quickly reach their target, but the fact is that ships face the targets they are firing upon, so by angling the gun ports outward you're actually increasing the travel time for any missiles to reach their targets under normal circumstances while only providing a slim benefit to the initial part of combat where a ship may not be facing its target.
7) I simply am not convinced that this focus on "Well it's a city-ship" means that we should be gimping the Bactrian's combat capabilities, either in reducing its customization or simply creating suboptimal firing angles for typical combat scenarios. The fact that it is a city-ship actually means, as the description states, that it is *designed* to be a *capable* warship, which it certainly is now. But the fact that it isn't a *dedicated* warship should not mean that we nerf its combat capabilities, especially seeing as a dedicated heavy warship is fully capable of taking on a Bactrian in combat; the Cruiser, Carrier, Dreadnought, and Kestrel as the other high-end human heavy warships are all able to take on a Bactrian, so among the high-end human heavy warships the Bactrian is *already at the bottom of the list as is in terms of combat*, and even low-end heavy warships like the Vanguard, Falcon, and Leviathan can take on a Bactrian given the right fittings, although granted that's in a player vs AI scenario as opposed to a head on brawl like the AI likes to do, in which case the winner is always going to be the ship with the bigger guns and higher HP, so it's a no-brainer that the Bactrian would beat a smaller warship.

Comment (Zitchas) [reply data/human/ships.txt:345]: Thank you for the detailed response. I really appreciate it! I won't response immediately, but I'll think about it and respond.

Comment (Saugia) [reply data/human/ships.txt]: For the sake of everyone's sanity, and to work on getting the main feature of this PR, the models in, I have elected to not use angled gun ports (which was mentioned in this PR at a previous time if you're from the future.) If this is still desired, I strongly encourage those who wish it to do so in a separate PR, and tag this one if you so desire.

If you could wait for Zitchas's response, Derpy, before finalizing anything, that would be great, as I to want to hear his opinion as well. But once we're past this, I don't want to dwell on it that much longer.

**Comment by Zitchas** [comment]
Posting this in the main comment thread so it doesn't get lost if/when that sub-comment gets marked as resolved, which it probably can be at this point.

From my perspective, point 7 is the one that I'm most firm on: It *is* a city ship, that is the entirety of its reason for existing. Any well built ship *will* be designed to be successful. The description flat out states that there used to be many, but this was the one that had sufficient edge to keep on going when the rest were decommissioned. That, to me, suggests that it has every edge that its designers could wiggle out of its design. 

That being said, yes, this reduces the flexibility of the Bactrian's outfitting a bit. That is, as far as I'm concerned, a good thing. If these ports aren't being optimized for ideal protection of its vulnerable flanks and rear, those ports have no business being on the ship at all. If we had missile hardpoints as something different from gun hardpoints, I'd be here insisting that the Bactrian should have zero gunports. It would actually make sense for it to have *more* hardpoints, actually - but only if they're all missile ports and angled off towards the flanks and rear.

Anyway, we have a plethora of warships in this game. One more warship does nothing for fun or interest. The Bactrian isn't just another warship in the lore, and it shouldn't be just another warship now. That's a useless, and pointless, thing to have in the game. Not to mention completely overshadowed by, as you point out, so many other dedicated warships. Which, again, shouldn't be surprising, because it's not a warship. It's a combo freighter/transport that just happens to have enough weaponry and protection to qualify as one. What the Bactrian is, however, is toughest and deadliest transport/freighter in all of human space. It's pretty much the one freighter that needs no escort. If anyone has a cargo that they want to get somewhere securely? Bactrian. You have a conference of VIPs or senators holding a space-based conference? Put them in a Bactrian. No other transport will keep them safer. 

This is, quite literally, the idea of leaning *into* the ships's strengths instead of bemoaning the fact that it is leaning away from something it never really should have been that good at to begin with. There is a world of a difference between "enough of a warship to fend off pirates" and "ship built to go hunting down problems and ramming them in the face." The Bactrian is the freighter/transport that no pirate should ever want to see jumping in, because it'll annihilate them without even bothering to change course. 

As for point 6 - This is basically a relic of the fact that the Bactrian (and most other giant ships) turn **far** too fast for their size. Assuming they actually had the mass they should have, them turning to bring forward guns to bear should be about as comical as watching a US Navy Carrier trying to fire unguided torpedoes at a high speed missile boat that is harassing it. This is basically reinforcing the fact that heavy warships are currently setup to act like really big fighters. We keep talking about fixing combat and making it more diversified. Well, part of the solution is to stop setting up heavy ships like fighters. The Bactrian shouldn't be spinning around. Against light warships, it *should* be expecting that 95% of the fight it will be pointed somewhere other than at the light things flitting around it. So having missiles set up to get that spread *should* be something that benefits it far more than the off chance that someone is dumb enough to fly in front of it. Think of it as future-proofing. As our AI improves, as big ships get slower, stuff will be staying out from in front of it more and more, and those forward guns will get more and more useless. And side-angled guns will become more and more valuable.

Anyway, I just want to re-iterate my thanks that you actually explained your position. Given the fact that we are continuing to try to encourage more people to do reviews, it is very helpful if you would actually make a practice of posting all your points. That helps people see what your thought processes are, and what sort of things to look for. It gives us all a chance to learn instead of just fuming at the fact that the black box spat out a pass/fail. 

From a community standpoint, the less we rely on "what you feel is best" and the more we can understand your decisions, the better off we will all be.

While I continue to disagree with the guns going straight forward, I don't consider my opinion here to be a blocking issue. Please proceed with merging whichever version you feel is best. My "approve" that I put on here earlier stands, regardless of which version is merged.

**Comment by kestrel1110** [comment]
Just a thought after reading Zitchas' thing- if we're gonna make heavy warships really slow turners, we're gonna have to find some way to compensate, and the only logical solution would be to begin implementing a series of very large reverse thrusters, and start putting them on virtually every ship. However, reverse thrusters right now use weapon space, as they are mounted 'in the forward sections of the ship' and that would make it slightly broken, as you would have to increase the general weapons capacity of every ship in order to fit them. However, that creates the problem of some players removing all of their reverse thrusters in order to include ridiculously large weapons, for example, we'd start seeing weapon variant kestrels with a full suite of ion cannons, or other broken things like that. So in my opinion, the best solution would be to make reverse thrusters utilize something other than weapon space, perhaps 'forward engine space' or something of that sort, and have each ship have a varying amount, increasing for each ship of a higher mass. However, this would by no means be a simple solution, and would require lots of effort, and should probably be shelved for a much later version.

**Comment by vitalchip** [comment]
I'm wading in late here but I think the angled gun ports are the right way to go. In the absence of dedicated missile ports this is a really good idea. This ship is massive (or supposed to be) it's got no business having fixed guns which would require a change of course to utilise. The only reason fixed guns exist on ES "big" ships is because they aren't "big" statistically and can turn far too fast.

**Comment by McloughlinGuy** [comment]
Fun fact: This pull request has the most likes AND dislikes out of any pull request. 2 dislikes is most, PR doesn't get dislikes much. Good design!

File: data/human/ships.txt [hunk 3]
```diff
@@ -333,21 +333,21 @@ ship "Bactrian"
...
+	turret 37 -132.5 "Heavy Anti-Missile Turret"
+	turret -39 -39 "Heavy Laser Turret"
+	turret 31.5 -26 "Heavy Laser Turret"
+	turret -42.5 75 "Heavy Laser Turret"
+	turret 32.5 132 "Heavy Laser Turret"
+	turret -33.5 192.5 "Heavy Anti-Missile Turret"
+	gun -9 -231.5 "Torpedo Launcher"
+	gun 13.5 -231.5 "Torpedo Launcher"
+	gun 36.5 -197 "Sidewinder Missile Launcher"
+	gun 42 -193 "Sidewinder Missile Launcher"
```
Comment (Amazinite) [inline data/human/ships.txt:347]:
```suggestion
	gun -9 -231.5 "Torpedo Launcher"
	gun 13.5 -231.5 "Torpedo Launcher"
	gun 36.5 -197 "Sidewinder Missile Launcher"
	gun 42 -193 "Sidewinder Missile Launcher"
	turret 37 -132.5 "Heavy Anti-Missile Turret"
	turret -39 -39 "Heavy Laser Turret"
	turret 31.5 -26 "Heavy Laser Turret"
	turret -42.5 75 "Heavy Laser Turret"
	turret 32.5 132 "Heavy Laser Turret"
	turret -33.5 192.5 "Heavy Anti-Missile Turret"
```

File: data/human/ships.txt [hunk 4]
```diff
@@ -333,21 +333,21 @@ ship "Bactrian"
...
+	bay "Fighter" -43.5 -125
+	bay "Fighter" -50 103.5
+	bay "Fighter" 53 -72.5
```
Comment (Amazinite) [inline data/human/ships.txt:350]:
```suggestion
	bay "Fighter" -43.5 -125
	bay "Fighter" 53 -72.5
	bay "Fighter" -50 103.5
```

File: data/human/ships.txt [hunk 5]
```diff
@@ -2420,15 +2420,15 @@ ship "Mule"
...
-	turret -9 59 "Heavy Anti-Missile Turret"
-	bay "Fighter" -16 -9
+	engine 11 128
+	engine -8 128
+	turret 19.5 -39 "Heavy Anti-Missile Turret"
+	turret -22 -53.5 "Heavy Laser Turret"
+	turret 20 33 "Heavy Laser Turret"
+	turret -20.5 103.5 "Heavy Anti-Missile Turret"
+	gun -10.5 -125 "Sidewinder Missile Launcher"
+	gun 11 -125 "Sidewinder Missile Launcher"
```
Comment (Amazinite) [inline data/human/ships.txt:2430]:
```suggestion
	gun -11 -125 "Sidewinder Missile Launcher"
	gun 11 -125 "Sidewinder Missile Launcher"
	turret -22 -53.5 "Heavy Laser Turret"
	turret 19.5 -39 "Heavy Anti-Missile Turret"
	turret 20 33 "Heavy Laser Turret"
	turret -20.5 103.5 "Heavy Anti-Missile Turret"
```

**Review by Amazinite — CHANGES_REQUESTED** [review]

**Review by Amazinite — APPROVED** [review]

**Comment by Zitchas** [comment]
This PR originally had the Bactrian and Mule being given angled gun ports. The reasons this make sense are myriad, especially from a "spread out the missiles and give them a head start for turning around to hit fighters attacking from behind" perspective.

While the issue was basically vetoed, I just thought I'd tack this on after the fact: I just checked the discord poll room, and noticed that the angled gun ports on the Bactrian poll was still up. So, after about  nine months of polling, the community vote totals stand at having the Bactrian have:
- Angled gun ports: 422
- Straight ahead gun ports: 44

So, 422 vs 44 in favor of the angled gun ports...

2022-Sep-05 Angled Bactrian gunports
